Gemfile

```diff
@@ -257,7 +257,7 @@ group :ui_dependencies do # Added to Bundler.require in config/application.rb
   manageiq_plugin "manageiq-decorators"
   manageiq_plugin "manageiq-ui-classic"
   # Modified gems (forked on Github)
-  gem "jquery-rjs",                     "=0.1.1.2",          :source => "https://rubygems.manageiq.org"
+  gem "jquery-rjs",                     "=0.1.1.2", :git => "https://github.com/jrafanie/jquery-rjs.git", :branch => "jquery_rjs_0_1_1_ruby3"
```

Comment:
This is the last todo item:  needs review/merge/new release https://github.com/ManageIQ/jquery-rjs/pull/9